tasks/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ add_dependencies(${ROCK_GAZEBO_TASKLIB_NAME}
 
 TARGET_LINK_LIBRARIES(${ROCK_GAZEBO_TASKLIB_NAME}
     ${OrocosRTT_LIBRARIES}
-    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES})
+    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES}
+    thruster_types)
```

Comment:
I would just include the protobufs in the main library instead of creating a separate one.

The easiest way to add dependencies, when using pkg-config, is to add

~~~
using_library 'name_of_pkgconfig_package'
~~~

in the orogen file.